Game/Saved/Logs/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
Why don't we edit the main git ignore and get rid of this one?

Reply:
Oh that makes sense I think :).

Reply:
I think the point of this file was to make sure there was _something_ in this directory so the directory gets created on disk, right?

Reply:
It sounds wrong to have to put in a file to this directory. I'm pretty sure unreal generates this folder when running. Do we have more details on what step in the setup process that generates this file. It shouldn't be required.

Reply:
Right clicking uproject and clicking generate project simply does not work on a fresh clone.